liblastlog2/src/liblastlog2.sym

```diff
@@ -1,4 +1,4 @@
-LIBLASTLOG2_1.0 {
+LIBLASTLOG2_2.0 {
```

Comment:
For the symbols versioning, we use the package version; see, for example, the same file for libmount.

Reply:
I have set it to LIBLASTLOG2_2_40